apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -599,19 +620,11 @@ const DanceAiModal: React.FunctionComponent = () => {
         <div id="buttons-area-top" className={moduleStyles.buttonsAreaTop}>
           {mode === Mode.RESULTS && (
             <div>
-              <Button
-                id="regenerate"
-                onClick={handleRegenerateClick}
-                color={Button.ButtonColor.white}
-                className={classNames(moduleStyles.button)}
-              >
-                <i className="fa fa-refresh" />
-              </Button>
               <Button
                 id="explanation-button"
                 text={'?'}
```

Comment:
Does this need to be localized as well?

Reply:
Good question.  We have the full string for this control ready to use, and if we stuck with a question mark, I assume we'd move to an icon, so I think we're okay for now.